.github/workflows/react_native_release.yaml

```diff
@@ -1,5 +1,5 @@
 # For details see: https://superface.ai/blog/npm-publish-gh-actions-changelog
-name: Release package
+name: React-Native Release package
```

Comment:
how do we release android package?
do we release all packages (rn and android) at once with the same version or separately?

Reply:
With jitpack all is necessary is tag. This workflow creates new tag so when react native is released then new tag is available for andoid jitpack. One thing that should be done is to somewhere in docks update to new tag.